### PR TITLE
data: ashutoshkandpal

### PR DESCRIPTION
**Pull request by ashutoshkandpal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4867"></a>

### CODE REVIEW

File: data/ashutoshkandpal.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			"icon": "FaInstagram"
+		},
+		{
+			"name": "Follow me on Linkedin",
+			"url": "https://www.linkedin.com/in/ashutoshkandpal/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Github",
+			"url": "https://www.linkedin.com/in/ashutoshkandpal/",
```
Comment (imbeshat) [inline data/ashutoshkandpal.json:22]: Should be your github url.

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by ashutoshkandpal — COMMENTED** [review]
updated link of github

File: data/ashutoshkandpal.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"name": "Ashutosh Kandpal",
+	"type": "personal",
+	"bio": "Technology Consultant and Social Entrepreneur, Yoga Certified Practitioner, Passionate about Travelling and Tech",
+	"links": [{
+			"name": "Twitter",
+			"url": "https://twitter.com/SaraJaoude",
```
Comment (imbeshat) [inline data/ashutoshkandpal.json:7]: Your Twitter link

File: data/ashutoshkandpal.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Follow me on Instagram",
+			"url": "https://www.instagram.com/eddieandsaraexplore/",
```
Comment (imbeshat) [inline data/ashutoshkandpal.json:12]: Your Instagram link

**Review by imbeshat — CHANGES_REQUESTED** [review]
Please, update these links also

**Review by imbeshat — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**